applications/ConvectionDiffusionApplication/custom_strategies/strategies/explicit_runge_kutta_4_eulerian_convdiff_strategy.h

```diff
@@ -0,0 +1,385 @@
+    {
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(
+        ModelPart &rModelPart,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : BaseType(rModelPart, MoveMeshFlag, RebuildLevel)
+    {
+    }
+
+    /** Copy constructor.
+     */
+    ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(const ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion &Other) = delete;
+
+    /** Destructor.
+     */
+    virtual ~ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion() = default;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /**
+     * @brief Initialization of variables.
+     * @details In this method, we call the base strategy initialize and initialize the projection variable.
+     * This is required to prevent OpenMP errors as the projection variable is stored in the non-historical database.
+     */
+    void Initialize() override
+    {
+        KRATOS_TRY;
+
+        auto& r_model_part = BaseType::GetModelPart();
+        auto& r_process_info = r_model_part.GetProcessInfo();
+        // Call the base method
+        BaseType::Initialize();
+        // If required, initialize the OSS projection variables
+        if (r_process_info[OSS_SWITCH]) {
+            for (auto& r_node : r_model_part.GetCommunicator().LocalMesh().Nodes()) {
+                r_node.SetValue(SCALAR_PROJECTION, 0.0);
+            }
+        }
+        r_process_info.GetValue(TIME_INTEGRATION_THETA) = 1.0;
+
+        KRATOS_CATCH("");
+    }
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    /// Turn back information as a string.
+    std::string Info() const override
+    {
+        return "ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion";
+    }
+
+    /// Print information about this object.
+    void PrintInfo(std::ostream &rOStream) const override
+    {
+        rOStream << Info();
+    }
+
+    /// Print object's data.
+    void PrintData(std::ostream &rOStream) const override
+    {
+        rOStream << Info();
+    }
+
+    ///@}
+
+protected:
+    ///@name Protected static Member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Protected member Variables
+    ///@{
+
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+    /**
+     * @brief Initialize the Runge-Kutta substep.
+     * @details Calculate the orthogonal subscale projections if required.
+     */
+    virtual void InitializeRungeKuttaIntermediateSubStep() override
+    {
+        KRATOS_TRY;
+
+        BaseType::InitializeRungeKuttaIntermediateSubStep();
+
+        auto& r_model_part = BaseType::GetModelPart();
+        auto& r_process_info = r_model_part.GetProcessInfo();
```

Comment:
```suggestion
        const auto& r_process_info = r_model_part.GetProcessInfo();
```

Reply:
Since I modify `TIME_INTEGRATION_THETA` value, it cannot be `const`.